### PR TITLE
OWNERS.md: add events project area

### PR DESCRIPTION
**Pull request by Rugvip**
New incubating Project Area for the Events system 🎉

It's basically all built by @pjungermann and he's kept helping out to maintain and evolve it since its introduction ❤️ 

An open question here is whether the modules should be included in the scope of the area or not. I kept them in because you where listed as codeowner for them @pjungermann, but happy to keep them out of scope too.

@pjungermann @backstage/maintainers please approve if this all looks good

### CODE REVIEW

File: plugins/events-backend-module-aws-sqs/catalog-info.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@ metadata:
 spec:
   lifecycle: experimental
   type: backstage-backend-plugin-module
-  owner: pjungermann
+  owner: maintainers
```
Comment (freben) [inline plugins/events-backend-module-aws-sqs/catalog-info.yaml:9]: all of these should be events-maintainers right

Comment (Rugvip) [inline plugins/events-backend-module-aws-sqs/catalog-info.yaml:9]: Hmm, will suggest a fix for this separately

Comment (Rugvip) [reply plugins/events-backend-module-aws-sqs/catalog-info.yaml:9]: @freben ^, it's what the script does atm

**Review by freben — APPROVED** [review]
alright sure then

**Review by pjungermann — APPROVED** [review]
Thank you ❤️

**Pull request merged**